src/tests/features/rich_tcp_mss_clamp.at

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<zone>
+  <service name="ssh"/>
+  <service name="dhcpv6-client"/>
+  <rule>
+    <tcp-mss-clamp value="pmtu"/>
+    <accept/>
+  </rule>
+</zone>
+])
+FWD_CHECK([--check-config],0, ignore)
+
+AT_DATA([./zones/tcp.xml], [dnl
+<?xml version="1.0" encoding="utf-8"?>
+<zone>
+  <service name="ssh"/>
+  <service name="dhcpv6-client"/>
+  <rule>
+    <tcp-mss-clamp value="thisdoesnotexist"/>
+    <accept/>
+  </rule>
+</zone>
+])
+m4_ifdef([TESTING_FIREWALL_OFFLINE_CMD], [FWD_CHECK([--check-config], 0, ignore, [dnl
+WARNING: INVALID_RULE: thisdoesnotexist: rule tcp-mss-clamp value="thisdoesnotexist" accept
+WARNING: INVALID_RULE: thisdoesnotexist: rule tcp-mss-clamp value="thisdoesnotexist" accept
+])], [FWD_GREP_LOG([dnl
+WARNING: INVALID_RULE: thisdoesnotexist: rule tcp-mss-clamp value="thisdoesnotexist" accept
+])])
+
+AT_DATA([./zones/tcp.xml], [dnl
+<?xml version="1.0" encoding="utf-8"?>
+<zone>
+  <service name="ssh"/>
+  <service name="dhcpv6-client"/>
+  <rule>
+    <tcp-mss-clamp/>
+    <accept/>
+  </rule>
+</zone>
+])
+m4_ifdef([TESTING_FIREWALL_OFFLINE_CMD], [FWD_CHECK([--check-config], 2, ignore,ignore)], 
+[FWD_CHECK([--check-config],254, ignore, ignore)])
+
+dnl start cli tests (permanent configuration)
+AT_DATA(rm ./zones/tcp.xml)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=thisdoesnotexist accept'],122,ignore,ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp accept'],122,ignore, ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=0 accept'],122,ignore, ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=536 accept'],0,ignore, ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=pmtu accept'],0,ignore, ignore)
+
+dnl start runtime tests for tcp/mss clamp
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp value=pmtu accept'],0,ignore, ignore)
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp value=0 accept'],122,ignore, ignore)
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp accept'],122,ignore, ignore)
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp value=536 accept'],0,ignore, ignore)
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp value=thisdoesnotexist accept'],122,ignore,ignore)
+
+NFT_LIST_RULES([inet], [filter_FWDO_public_allow], 0, [dnl
+        table inet firewalld {
+        chain filter_FWDO_public_allow {
+        tcp flags syn tcp option maxseg size set rt mtu accept
+        tcp flags syn tcp option maxseg size set 536 accept
+        }
+        }
+    ])
+
+IPTABLES_LIST_RULES([filter], [FWDO_public_allow], 0, [dnl
+    TCPMSS tcp -- 0.0.0.0/0 0.0.0.0/0 tcp flags:0x06/0x02 TCPMSS clamp to PMTU
+    TCPMSS tcp -- 0.0.0.0/0 0.0.0.0/0 tcp flags:0x06/0x02 TCPMSS set 536
+    ])
+
+IP6TABLES_LIST_RULES([filter], [FWDO_public_allow], 0, [dnl
+    TCPMSS tcp ::/0 ::/0 tcp flags:0x06/0x02 TCPMSS clamp to PMTU
+    TCPMSS tcp ::/0 ::/0 tcp flags:0x06/0x02 TCPMSS set 536
+    ])
+
```

Comment:
Please add negative tests to make sure `tcp-mss-clamp` with an aciton, e.g. `accept` is rejected.